meson_options.txt

```diff
@@ -12,11 +12,13 @@ option('bluez', type : 'boolean', value : false, description : 'enable BlueZ sup
 option('polkit', type: 'boolean', value : true, description : 'enable PolKit support in daemon')
 option('gnutls', type: 'boolean', value : true, description : 'enable GnuTLS support')
 option('lzma', type: 'boolean', value : false, description : 'enable LZMA support')
+option('libprotobuf-c', type : 'boolean', value : true, description : 'enable protobuf-c support')
```

Comment:
I don't see a strong reason to make this a separate option AND to make the new plugin an option.  It should be either or (and I would lean on plugin only)